schemas/webpack.d.ts

```diff
@@ -0,0 +1,370 @@
+        pathinfo?: boolean;
+
+        // The `publicPath` specifies the public URL address of the output files when referenced in a browser.
+        publicPath?: string;
+
+        // The filename of the SourceMaps for the JavaScript files. They are inside the `output.path` directory.
+        sourceMapFilename?: string;
+
+        // Prefixes every line of the source in the bundle with this string.
+        sourcePrefix?: string;
+
+        // If `output.libraryTarget` is set to umd and `output.library` is set, setting this to true will name the AMD module.
+        umdNamedDefine?: boolean;
+    }
+
+    interface OutputAuxiliaryComment {
+        amd?: string;
+        commonjs?: string;
+        commonjs2?: string;
+        root?: string;
+    }
+
+    interface OutputDevtoolLineToLine {
+        exclude?: string;
+        include?: string;
+        test?: string;
+    }
+
+    interface Resolve {
+        alias?: { [key: string]: string } | ResolveAlias[];
+        aliasFields?: [string | string[]];
+        cachePredicate?: Function;
+        descriptionFiles?: string[];
+        enforceExtension?: boolean;
+        enforceModuleExtension?: boolean;
+        extensions?: string[];
+        fileSystem?: any; // TODO missing type
+        mainFields?: [string | string[]];
+        mainFiles?: string[];
+        moduleExtensions?: string[];
+        modules?: string[];
+        plugins?: any[];
+        resolver?: any; // TODO missing type
+        symlinks?: boolean;
+        unsafeCache?: boolean | Object;
+    }
+
+    interface ResolveAlias {
+        alias?: string;
+        name?: string;
+        onlyModule?: boolean;
+    }
+
+    interface Object {
+        [key: string]: any;
+    }
+
+    type EntryItem = string | string[];
+
+    type OutputLibraryTarget =
+        "var"
+        | "assign"
+        | "this"
+        | "window"
+        | "global"
+        | "commonjs"
+        | "commonjs2"
+        | "commonjs-module"
+        | "amd"
+        | "umd"
+        | "umd2"
+        | "jsonp";
+
+    type Stats =
+        "none"
+        | "errors-only"
+        | "minimal"
+        | "normal"
+        | "verbose";
+
+    type Target =
+        "web"
+        | "webworker"
+        | "node"
+        | "async-node"
+        | "node-webkit"
+        | "atom"
+        | "electron"
+        | "electron-main"
+        | "electron-renderer";
+
+    type Externals = string | Object | ((context: any, request: any, callback: (err: any, result: any)=> any) => any) | RegExp;
+
+    type RuleSetConditionSimple = RegExp | string | Function | RuleSetConditionObject;
+    type RuleSetCondition = RuleSetConditionSimple | RuleSetConditionSimple[];
+    type RuleSetConditions = RuleSetCondition[];
+
+    type RuleSetUse = RuleSetUseItem | RuleSetUseItem[];
+    type RuleSetUseItem = string | RuleSetUseItemObject;
+
+    type RuleSetQuery = Object | string;
+
+    type RuleSetRules = RuleSetRule[];
+}
+
+interface Webpack {
```

Comment:
This interface should be exported, no? Doesn't make much sense having a type definition otherwise.